include/z64.h

```diff
@@ -334,6 +334,17 @@ typedef struct {
     /* 0x01E2 */ char unk_1E2[0x06];
 } ConsoleLogoState; // size = 0x1E8
 
+#if IS_MAP_SELECT_ENABLED
```

Comment:
``IS_DEBUG_BOOT_ENABLED``? also it's probably worth to move the structs/functions declarations to a dedicated header